src/NuGet.Core/NuGet.Packaging/Core/PackageDependencyGroup.cs

```diff
@@ -18,6 +19,21 @@ public class PackageDependencyGroup : IEquatable<PackageDependencyGroup>, IFrame
         private readonly NuGetFramework _targetFramework;
         private readonly IEnumerable<PackageDependency> _packages;
 
+        [JsonConstructor]
+        private PackageDependencyGroup(NuGetFramework targetFramework)
```

Comment:
I don't understand how the JSON deserialization sets the packages list, if this is the json constructor. But there are tests that pass? 😕